content/docs/services/intro.md

```diff
@@ -5,12 +5,36 @@ menu:
 aliases: 
     - /docs/services/
 title: Introduction
-layout: services
 weight: -1
 ---
 
-cloud.gov offers multiple [managed services]({{< relref "docs/apps/managed-services.md" >}}) to allow your application to expand its functionality. You can also create [user-provided services]({{< relref "docs/apps/production-ready.md#protect-access-to-sensitive-credentials" >}}).
+You can expand the functionality of your cloud.gov application by making use of services. Before your application can use a service, the service must be provisioned, and credentials for using the service must be supplied to your application. 
 
-To list all the managed services and plans available to your organization, you can run `cf marketplace` from your command line.
+There are two ways to provision services:
+
+- You can provision a service manually outside of cloud.gov, then create a [user-provided service instance](https://docs.cloudfoundry.org/devguide/services/user-provided.html) to hold the credentials.
+- You can provision a [managed service instance](https://docs.cloudfoundry.org/devguide/services/#instances) through the marketplace in cloud.gov on demand.
+
+In both cases, you make the credentials available by [binding the service instance to your application](https://docs.cloudfoundry.org/devguide/services/application-binding.html). 
+
+## Marketplace services
+cloud.gov offers a marketplace of FedRAMP-authorized [managed services]({{< relref "docs/apps/managed-services.md" >}}) that we operate in a secure and compliant manner on your behalf. You can also [extend the marketplace](#extending-the-marketplace) to include additional services run by other organizations.
+
+To list all the managed services and plans available to a given space, you run `cf marketplace` from your command line. Here is a list of the managed services that are generally available: 
+
+{{% services-table %}}
+
+## Extending the marketplace
+
+"Brokers" are the invisible integrators that enable you to set up managed service instances in cloud.gov in a consistent and self-service fashion. A broker offers a simple API that manages the service instance lifecycle. You can run your own broker to make a service from outside cloud.gov available through cloud.gov's marketplace. 
+
+*Note that when you extend the cloud.gov marketplace with your own broker, the cloud.gov team cannot vouch for the security or compliace of the brokered services. You should consider the security and compliance impact of the services you broker yourself.*
```

Comment:
"You should consider the security and compliance impact of the services you broker yourself." -> "You will need to document and authorize your own brokered services according to your own agency compliance requirements."